processor/transformer/transformer_test.go

```diff
@@ -48,6 +49,36 @@ func (t *fakeTransformer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func TestTransformerTimeoutAbort(t *testing.T) {
+	config.Reset()
+	logger.Reset()
+	config.Set("HttpClient.procTransformer.timeout", 1)
```

Comment:
Is it possible to do:
```suggestion
	config.Set("HttpClient.procTransformer.timeout", "1ms")
```

Reply:
damn... I didn't know we can do something like this. Thanks, modified.